tinyagentos/routes/store_install.py

```
@@ -1,18 +1,154 @@
+    if not target_remote:
+        return "127.0.0.1"
+    try:
+        import tinyagentos.containers as containers
+        remotes = await containers.remote_list()
+        for r in remotes:
+            if r.get("name") == target_remote:
+                addr = r.get("addr", "")
+                parsed = urlparse(addr)
+                if parsed.hostname:
+                    return parsed.hostname
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)